src/main/java/ch/njol/skript/effects/EffToggle.java

```diff
@@ -45,16 +48,20 @@
  */
 @SuppressWarnings("deprecation")
```

Comment:
Can you move this to the related methodd instead?

Reply:
Didn't move this annotation